lib/yabeda/activejob/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Yabeda
   module ActiveJob
-    VERSION = "0.5.0"
+    VERSION = "0.5.1"
```

Comment:
We'll have to bump this to 0.6.0 as I adjusted the minimum Ruby / Rails version in the previous PR.